apigee/adapter.go

```diff
@@ -410,18 +387,22 @@ func (h *handler) HandleQuota(ctx context.Context, inst *quotaT.Instance, args a
 // For future compatibility with Istio, accepts the "encoded" base64 string value
 // until request.auth.claims attribute is defined and supported.
 // see: https://github.com/istio/istio/issues/3194
-func convertClaims(log adapter.Logger, claims map[string]string) map[string]interface{} {
-	var claimsOut = map[string]interface{}{}
-	if len(claims) > 1 {
-		for k, v := range claims {
-			claimsOut[k] = v
+func resolveClaims(log adapter.Logger, claimsIn map[string]string) map[string]interface{} {
```

Comment:
Can you add a little comment to say what this function does? It looks like it's filtering the claims, but I am not 100% sure.

Reply:
done. does it make sense?

Reply:
Yup LGTM.